scripts/datamodel_generation.py

```diff
@@ -17,24 +17,30 @@
 
 import datamodel_code_generator.model.pydantic
 from datamodel_code_generator.imports import Import
+import os
+
 
-UNICODE_REGEX_REPLACEMENT_FILE_PATHS = [
-    "ingestion/src/metadata/generated/schema/entity/classification/tag.py",
-    "ingestion/src/metadata/generated/schema/entity/events/webhook.py",
-    "ingestion/src/metadata/generated/schema/entity/teams/user.py",
-    "ingestion/src/metadata/generated/schema/entity/type.py",
-    "ingestion/src/metadata/generated/schema/type/basic.py",
-]
 
 datamodel_code_generator.model.pydantic.types.IMPORT_SECRET_STR = Import.from_full_path(
     "metadata.ingestion.models.custom_pydantic.CustomSecretStr"
 )
 
 from datamodel_code_generator.__main__ import main
 
-args = "--input openmetadata-spec/src/main/resources/json/schema --input-file-type jsonschema --output ingestion/src/metadata/generated/schema --set-default-enum-member".split(
-    " "
-)
+current_directory = os.getcwd()
+ingestion_path = "./" if current_directory.endswith("OpenMetadata/ingestion") else "ingestion/"
+directory_root = "../" if current_directory.endswith("OpenMetadata/ingestion") else "./"
```

Comment:
@TomBushell chances are the directory would be named as something else for different users, we could keep `endswith("/ingestion")` to determine the ingestion directory, what do you think about that?

Reply:
Yep, this will work.